test/features/inventory_of_diffs.feature

```diff
@@ -0,0 +1,18 @@
+# Date:  08/07/2020
+# Author: Eugeniu Costetchi
+# Email: costezki.eugen@gmail.com
+
+Feature: List the diffs in the triplestore
```

Comment:
![image](https://github.com/jneubert/skos-history/wiki/images/skos-history3.png)
are these what this feature is about?

Reply:
exactly. We will have to discover how exactly that query will look like. That would be my job ;)